ui/react-components/components/GridSummary/GridSummary.jsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { injectIntl } from "react-intl";
+import "./GridSummary.module.scss";
+import moment from "moment";
+import { sortBy } from "lodash";
+import classNames from 'classnames'
+
+const transformGridData=(gridData)=>{
+  return gridData.map(dataElement => {
+    let rowList = [];
+    for (let element in dataElement.appointmentCountMap) {
+      rowList.push({
+        date: element,
+        count: dataElement.appointmentCountMap[element].allAppointmentsCount,
+        missedCount:
+          dataElement.appointmentCountMap[element].missedAppointmentsCount
+      });
+    }
+    return {
+      rowLabel: dataElement.appointmentService.name,
+      rowDataList: sortBy(rowList, dateObj => moment(dateObj.date))
+    };
+  });
+}
+
+const GridSummary = props => {
+  const { gridData=[], weekStartDate = moment().startOf("isoweek"), onClick } = props;
+  let week = []
+  const gridSummaryData=transformGridData(gridData)
+
+  return (
+    <div className="table-grid-wrapper">
```

Comment:
lets use css class variables instead of directly using the css names

Reply:
Ok. will update